### PR TITLE
Switch from URL prefixes to host names

### PR DESCRIPTION
**Pull request by votdev**
- Switch back to 'Traefik' as default Ingress because it is shipped via k3s and the S3 auth issues have been solved.
- Increase wait delay. Took a little bit longer on my system.
- Support SSL for Traefik ingresses.
- Adapt s3cmd config file. It should work with Traefik and NGINX ingresses.

Signed-off-by: Volker Theile <vtheile@suse.com>

### CODE REVIEW

**Comment by giubacc** [comment]
@votdev First of all, excuse me for pushing directly on your PR, that was not the initial intention.
I was trying to suggest some changes, but I'm still not completely familiar with github interface for managing PRs, so I did an unintentional commit, messing up things.
At that point I thought that the best thing to do, would had been to force-push the cleaned commits to fix the PR.
If you prefer me to open a separate PR based on this, please, force push on your latest commit.
My commits are basically some fixes for the vm based installation.
I tested your work and Traefik is working good with host-based ingresses.
I added a NodePort service pointing to the ingress because a FQDN address would work only if the k8s cluster has 1 node.
Having a NodePort allows to externally access an application independently of where physically the ingress controller is allocated in the cluster. 
This at the price to allocate a 3XXXX port on every node of the cluster.
I also added a plain http ingress for s3gw application (to test the S3 explorer).

File: env/ingress-traefik/s3gw-ingress.yaml [hunk 1]
```diff
@@ -39,3 +39,23 @@ spec:
                 name: s3gw-service
                 port:
                   number: 80
+---
+apiVersion: networking.k8s.io/v1
```
Comment (votdev) [inline env/ingress-traefik/s3gw-ingress.yaml:43]: This ingress is not needed, i can still reach HTTP via `s3gw.local` without the `s3gw-no-tls-ingress` object.

Comment (giubacc) [reply env/ingress-traefik/s3gw-ingress.yaml:43]: It's still not clear why, but from my tests it seems that an ingress without tls is needed to make S3 explorer working.
If I remove the no-tls ingress the S3 explorer stops working.

**Review by giubacc — APPROVED** [review]
LGTM

**Pull request merged**